pyproject.toml

```diff
@@ -11,7 +11,6 @@ show_missing = true
 [tool.pytest.ini_options]
 minversion = "6.0"
 log_cli_level = "INFO"
-asyncio_mode = "auto"
```

Comment:
Iirc, removing this would fail the itests.

Reply:
not any longer. locally they pass, and the previous CI runs only the integration tests were red.